iOS/Flows/Conversation/ConversationListCoordinator.swift

```diff
@@ -13,44 +13,50 @@ import PhotosUI
 import Combine
 import StreamChat
 
-class ConversationCoordinator: PresentableCoordinator<Void> {
+class ConversationListCoordinator: PresentableCoordinator<Void> {
 
-    lazy var conversationVC = ConversationViewController(conversation: ConversationsManager.shared.activeConversations.last,
-                                                         startingMessageId: self.startingMessageId)
+    lazy var conversationListVC = ConversationListViewController(members: self.conversationMembers)
 
-    let startingMessageId: MessageId?
+    private let conversationMembers: [ConversationMember]
+    private let startingConversationID: ConversationID?
 
     override func toPresentable() -> DismissableVC {
-        self.conversationVC.onSelectedThread = { [unowned self] (channelID, messageID) in
-            self.presentThread(for: channelID, messageID: messageID)
-        }
-        return self.conversationVC
+        return self.conversationListVC
     }
 
     init(router: Router,
          deepLink: DeepLinkable?,
-         conversation: Conversation?,
-         startingMessageId: MessageId?) {
+         conversationMembers: [ConversationMember],
+         startingConversationID: ConversationID?) {
```

Comment:
This PR doesn't actually auto scroll to the starting conversation. Work on that is forthcoming.